mods/ctf/ctf_map/ctf_traps.lua

```diff
@@ -193,12 +197,41 @@ core.register_node("ctf_map:damage_cobble", {
 			return
 		end
 
-		if not damage_cobble_dig(pos, node, shooter) then
+		if not damage_node_dig(pos, node, shooter) then
+			return core.dig_node(pos)
+		end
+	end,
+	on_dig = function(pos, node, digger)
+		if not damage_node_dig(pos, node, digger) then
+			return core.node_dig(pos, node, digger)
+		end
+	end,
+	after_place_node = function(pos, placer, itemstack, pointed_thing)
+		local meta = core.get_meta(pos)
+		meta:set_string("placer", placer:get_player_name())
+	end,
+})
+
+core.register_node("ctf_map:damage_glass", {
+	description = "Damage Glass",
+	tiles = { "default_glass.png^[colorize:#B00000:180^default_glass_detail.png", "default_glass_detail.png" },
+	groups = { cracky = 3 },
+	sounds = default.node_sound_glass_defaults(),
+	drawtype = "glasslike_framed_optional",
+	use_texture_alpha = "clip",
+	paramtype = "light",
+	sunlight_propagates = true,
+	on_ranged_shoot = function(pos, node, shooter, type)
+		if type == "pistol" then
+			return
+		end
```

Comment:
```suggestion
```

Reply:
As normal glass breaks with pistol, it makes sense that damage glass break, too.